amundsen_application/static/js/ducks/tableMetadata/api/v0.ts

```diff
@@ -173,13 +173,13 @@ export function updateColumnDescription(
   });
 }
 
-export function getLastIndexed() {
-  return axios
-    .get(`${API_PATH}/get_last_indexed`)
-    .then((response: AxiosResponse<LastIndexedAPI>) => {
-      return response.data.timestamp;
-    });
-}
+// export function getLastIndexed() {
+//   return axios
+//     .get(`${API_PATH}/get_last_indexed`)
+//     .then((response: AxiosResponse<LastIndexedAPI>) => {
+//       return response.data.timestamp;
+//     });
+// }
```

Comment:
Let's remove all commented code, thanks!

Reply:
yes I will be removing all the comments since those are related to old code.